### DISCUSSION TITLE
You are recommending me to follow a user that I have blocked

### DISCUSSION THREAD

**Discussion by ruuda** [community/community#13161] [Other Feature Feedback, Questions, & Ideas]
You are recommending me to follow a user that I have blocked. Obviously, I don’t want to follow this user. But the user is still displayed prominently in the feed.

**Comment by bdresser** [comment]
Hi @ruuda, thanks for the feedback and sorry to hear this! We have checks in place to make sure we don't recommend users the viewer has blocked — we're investigating your report that those checks aren't effective, but we are struggling to reproduce. 

Could you email me (bdresser at github dot com) with a screenshot of what you're seeing?

**Reply by ruuda** [reply]
Thanks for looking into this, I sent you a mail.

**Reply by paullouisageneau** [reply]
Same here, I sent you an email if it helps.

**Reply by chuigda** [reply]
It's 9, June, 2022, seems still not fixed.

**Reply by pedrosantosara** [reply]
bbb

**Comment by mikekavouras** [comment]
A (hopefully final) fix went live today. Please let us know if the problem persists.

**Reply by expenses** [reply]
This still happens. I see the feeds of people I've blocked.

**Reply by mikekavouras** [reply]
@expenses I've taken a look at it appears you're not blocking anyone. Would you mind double checking on your end? Feel free to reach out to support (mention my name) if you'd like to discuss information that's too sensitive for a public forum.

**Reply by expenses** [reply]
I was seeing the feeds of people i had blocked. I stopped blocking them as I realized that it didn't stop their feeds

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
